Added new bullets and links to Adopt section

jwayman opened the pull request:
Marketing request to add new bullets to adopt section with corresponding links.



## Test plan

Verify links and check langauge.

lcolston08 reviewed (changes requested):
Please hosted the two documents outside of Google Drive so they can be openly accessed. Thanks!

jwayman commented:
> Please hosted the two documents outside of Google Drive so they can be openly accessed. Thanks!

I've added these files to Google cloud, and updated the links.
